src/server/controllers/match.controller.js

```diff
@@ -165,6 +173,17 @@ export function getInitialMentorList(userId) {
           .sort({ stamp_login: -1 }).exec();
       })
       .then((user) => {
+        const userData = JSON.parse(JSON.stringify(user));
+        return new Promise((resolve) => {
+          userData.forEach(item => {
+            if (pendingList.includes(item._id.toString())) {
+              item.pending = true;
+            }
+          });
+          resolve(userData);
```

Comment:
I think this resolve can make issue.
Check this.
http://stackoverflow.com/questions/18983138/callback-after-all-asynchronous-foreach-callbacks-are-completed